src/java/fr/paris/lutece/plugins/workflow/modules/appointmentants/service/ITaskAntsAppointmentService.java

```diff
@@ -0,0 +1,14 @@
+package fr.paris.lutece.plugins.workflow.modules.appointmentants.service;
+
+import java.util.List;
+
+import org.springframework.transaction.annotation.Transactional;
+
+public interface ITaskAntsAppointmentService {
+
+	@Transactional( WorkflowAppointmentAntsPlugin.BEAN_TRANSACTION_MANAGER)
```

Comment:
Please use the transaction manager provided by lutece-core (TransactionManager.java) and remove the Spring annotation.